common/src/main/java/io/novafoundation/nova/common/utils/recyclerView/expandable/ExpandableAnimationSettings.kt

```diff
@@ -0,0 +1,5 @@
+package io.novafoundation.nova.common.utils.recyclerView.expandable
+
+import android.view.animation.Interpolator
+
+class ExpandableAnimationSettings(val duration: Long, val interpolator: Interpolator)
```

Comment:
Useful to share animation settings between expanding/collapsing animation and item animation